Subject: Fix BeeCrypt headers

The headers use #include "beecrypt/*.h" instead of #include <beecrypt/*.h>
which leads to recursive pathnames in the DWARF info of the objects.

Signed-off-by: Jan Blunck <jblunck@suse.de>
---
 aes.h         |    4 ++--
 aesopt.h      |    4 ++--
 api.h         |    6 +++---
 base64.h      |    2 +-
 beecrypt.h    |    6 +++---
 blockmode.h   |    2 +-
 blockpad.h    |    2 +-
 blowfish.h    |    4 ++--
 blowfishopt.h |    4 ++--
 dhaes.h       |    4 ++--
 dldp.h        |    2 +-
 dlkp.h        |    2 +-
 dlpk.h        |    2 +-
 dlsvdp-dh.h   |    4 ++--
 dsa.h         |    2 +-
 elgamal.h     |    2 +-
 endianness.h  |    2 +-
 entropy.h     |    2 +-
 fips186.h     |    6 ++----
 hmac.h        |    2 +-
 hmacmd5.h     |    4 ++--
 hmacsha1.h    |    4 ++--
 hmacsha256.h  |    4 ++--
 hmacsha384.h  |    4 ++--
 hmacsha512.h  |    4 ++--
 md5.h         |    2 +-
 memchunk.h    |    2 +-
 mp.h          |    4 ++--
 mpbarrett.h   |    4 ++--
 mpnumber.h    |    2 +-
 mpprime.h     |    2 +-
 mtprng.h      |    2 +-
 pkcs1.h       |    2 +-
 pkcs12.h      |    2 +-
 rsa.h         |    2 +-
 rsakp.h       |    2 +-
 rsapk.h       |    2 +-
 sha1.h        |    4 ++--
 sha1opt.h     |    4 ++--
 sha256.h      |    2 +-
 sha384.h      |    2 +-
 sha512.h      |    2 +-
 sha_k.h       |    2 +-
 timestamp.h   |    2 +-
 44 files changed, 64 insertions(+), 66 deletions(-)

Index: beecrypt-4.1.2/aes.h
===================================================================
--- beecrypt-4.1.2.orig/aes.h
+++ beecrypt-4.1.2/aes.h
@@ -26,8 +26,8 @@
 #ifndef _AES_H
 #define _AES_H
 
-#include "beecrypt/beecrypt.h"
-#include "beecrypt/aesopt.h"
+#include <beecrypt/beecrypt.h>
+#include <beecrypt/aesopt.h>
 
 /*!\brief Holds all the parameters necessary for the AES cipher.
  * \ingroup BC_aes_m
Index: beecrypt-4.1.2/aesopt.h
===================================================================
--- beecrypt-4.1.2.orig/aesopt.h
+++ beecrypt-4.1.2/aesopt.h
@@ -26,8 +26,8 @@
 #ifndef _AESOPT_H
 #define _AESOPT_H
 
-#include "beecrypt/beecrypt.h"
-#include "beecrypt/aes.h"
+#include <beecrypt/beecrypt.h>
+#include <beecrypt/aes.h>
 
 #ifdef __cplusplus
 extern "C" {
Index: beecrypt-4.1.2/api.h
===================================================================
--- beecrypt-4.1.2.orig/api.h
+++ beecrypt-4.1.2/api.h
@@ -31,9 +31,9 @@
 
 #if WIN32
 # if !__CYGWIN32__
-#  include "beecrypt/win.h"
+#  include <beecrypt/win.h>
 # else
-#  include "beecrypt/gnu.h"
+#  include <beecrypt/gnu.h>
 # endif
 # ifdef BEECRYPT_DLL_EXPORT
 #  define BEECRYPTAPI __declspec(dllexport)
@@ -48,7 +48,7 @@
 #  define BEECRYPTCXXTEMPLATE extern
 # endif
 #else
-# include "beecrypt/gnu.h"
+# include <beecrypt/gnu.h>
 # define BEECRYPTAPI
 # define BEECRYPTCXXAPI
 #endif
Index: beecrypt-4.1.2/base64.h
===================================================================
--- beecrypt-4.1.2.orig/base64.h
+++ beecrypt-4.1.2/base64.h
@@ -25,7 +25,7 @@
 #ifndef _BASE64_H
 #define _BASE64_H
 
-#include "beecrypt/beecrypt.h"
+#include <beecrypt/beecrypt.h>
 
 /*!\
  * Decode white space character set (default).
Index: beecrypt-4.1.2/beecrypt.h
===================================================================
--- beecrypt-4.1.2.orig/beecrypt.h
+++ beecrypt-4.1.2/beecrypt.h
@@ -30,10 +30,10 @@
 #ifndef _BEECRYPT_H
 #define _BEECRYPT_H
 
-#include "beecrypt/api.h"
+#include <beecrypt/api.h>
 
-#include "beecrypt/memchunk.h"
-#include "beecrypt/mpnumber.h"
+#include <beecrypt/memchunk.h>
+#include <beecrypt/mpnumber.h>
 
 /*
  * Entropy Sources
Index: beecrypt-4.1.2/blockmode.h
===================================================================
--- beecrypt-4.1.2.orig/blockmode.h
+++ beecrypt-4.1.2/blockmode.h
@@ -27,7 +27,7 @@
 #ifndef _BLOCKMODE_H
 #define _BLOCKMODE_H
 
-#include "beecrypt/beecrypt.h"
+#include <beecrypt/beecrypt.h>
 
 #ifdef __cplusplus
 extern "C" {
Index: beecrypt-4.1.2/blockpad.h
===================================================================
--- beecrypt-4.1.2.orig/blockpad.h
+++ beecrypt-4.1.2/blockpad.h
@@ -26,7 +26,7 @@
 #ifndef _BLOCKPAD_H
 #define _BLOCKPAD_H
 
-#include "beecrypt/beecrypt.h"
+#include <beecrypt/beecrypt.h>
 
 #ifdef __cplusplus
 extern "C" {
Index: beecrypt-4.1.2/blowfish.h
===================================================================
--- beecrypt-4.1.2.orig/blowfish.h
+++ beecrypt-4.1.2/blowfish.h
@@ -34,8 +34,8 @@
 #ifndef _BLOWFISH_H
 #define _BLOWFISH_H
 
-#include "beecrypt/beecrypt.h"
-#include "beecrypt/blowfishopt.h"
+#include <beecrypt/beecrypt.h>
+#include <beecrypt/blowfishopt.h>
 
 #define BLOWFISHROUNDS	16
 #define BLOWFISHPSIZE	(BLOWFISHROUNDS+2)
Index: beecrypt-4.1.2/blowfishopt.h
===================================================================
--- beecrypt-4.1.2.orig/blowfishopt.h
+++ beecrypt-4.1.2/blowfishopt.h
@@ -26,8 +26,8 @@
 #ifndef _BLOWFISHOPT_H
 #define _BLOWFISHOPT_H
 
-#include "beecrypt/beecrypt.h"
-#include "beecrypt/blowfish.h"
+#include <beecrypt/beecrypt.h>
+#include <beecrypt/blowfish.h>
 
 #ifdef __cplusplus
 extern "C" {
Index: beecrypt-4.1.2/dhaes.h
===================================================================
--- beecrypt-4.1.2.orig/dhaes.h
+++ beecrypt-4.1.2/dhaes.h
@@ -33,8 +33,8 @@
 #ifndef _DHAES_H
 #define _DHAES_H
  
-#include "beecrypt/beecrypt.h"
-#include "beecrypt/dldp.h"
+#include <beecrypt/beecrypt.h>
+#include <beecrypt/dldp.h>
 
 #ifdef __cplusplus
 struct BEECRYPTAPI dhaes_pParameters
Index: beecrypt-4.1.2/dldp.h
===================================================================
--- beecrypt-4.1.2.orig/dldp.h
+++ beecrypt-4.1.2/dldp.h
@@ -26,7 +26,7 @@
 #ifndef _DLDP_H
 #define _DLDP_H
 
-#include "beecrypt/mpbarrett.h"
+#include <beecrypt/mpbarrett.h>
 
 /*
  * Discrete Logarithm Domain Parameters - Prime
Index: beecrypt-4.1.2/dlkp.h
===================================================================
--- beecrypt-4.1.2.orig/dlkp.h
+++ beecrypt-4.1.2/dlkp.h
@@ -26,7 +26,7 @@
 #ifndef _DLKP_H
 #define _DLKP_H
 
-#include "beecrypt/dlpk.h"
+#include <beecrypt/dlpk.h>
 
 /*!\ingroup DL_m
  */
Index: beecrypt-4.1.2/dlpk.h
===================================================================
--- beecrypt-4.1.2.orig/dlpk.h
+++ beecrypt-4.1.2/dlpk.h
@@ -26,7 +26,7 @@
 #ifndef _DLPK_H
 #define _DLPK_H
 
-#include "beecrypt/dldp.h"
+#include <beecrypt/dldp.h>
 
 /*!\ingroup DL_m
  */
Index: beecrypt-4.1.2/dlsvdp-dh.h
===================================================================
--- beecrypt-4.1.2.orig/dlsvdp-dh.h
+++ beecrypt-4.1.2/dlsvdp-dh.h
@@ -26,8 +26,8 @@
 #ifndef _DLSVDP_DH_H
 #define _DLSVDP_DH_H
 
-#include "beecrypt/dldp.h"
-#include "beecrypt/dlkp.h"
+#include <beecrypt/dldp.h>
+#include <beecrypt/dlkp.h>
 
 #ifdef __cplusplus
 extern "C" {
Index: beecrypt-4.1.2/dsa.h
===================================================================
--- beecrypt-4.1.2.orig/dsa.h
+++ beecrypt-4.1.2/dsa.h
@@ -32,7 +32,7 @@
 #ifndef _DSA_H
 #define _DSA_H
 
-#include "beecrypt/dlkp.h"
+#include <beecrypt/dlkp.h>
 
 typedef dldp_p dsaparam;
 typedef dlpk_p dsapub;
Index: beecrypt-4.1.2/elgamal.h
===================================================================
--- beecrypt-4.1.2.orig/elgamal.h
+++ beecrypt-4.1.2/elgamal.h
@@ -38,7 +38,7 @@
 #ifndef _ELGAMAL_H
 #define _ELGAMAL_H
 
-#include "beecrypt/mpbarrett.h"
+#include <beecrypt/mpbarrett.h>
 
 #ifdef __cplusplus
 extern "C" {
Index: beecrypt-4.1.2/endianness.h
===================================================================
--- beecrypt-4.1.2.orig/endianness.h
+++ beecrypt-4.1.2/endianness.h
@@ -26,7 +26,7 @@
 #ifndef _ENDIANNESS_H
 #define _ENDIANNESS_H
 
-#include "beecrypt/beecrypt.h"
+#include <beecrypt/beecrypt.h>
 
 #if defined(__cplusplus) || HAVE_INLINE
 
Index: beecrypt-4.1.2/entropy.h
===================================================================
--- beecrypt-4.1.2.orig/entropy.h
+++ beecrypt-4.1.2/entropy.h
@@ -26,7 +26,7 @@
 #ifndef _ENTROPY_H
 #define _ENTROPY_H
 
-#include "beecrypt/beecrypt.h"
+#include <beecrypt/beecrypt.h>
 
 #if WIN32
 #include <windows.h>
Index: beecrypt-4.1.2/fips186.h
===================================================================
--- beecrypt-4.1.2.orig/fips186.h
+++ beecrypt-4.1.2/fips186.h
@@ -26,7 +26,8 @@
 #ifndef _FIPS186_H
 #define _FIPS186_H
 
-#include "beecrypt/beecrypt.h"
+#include <beecrypt/beecrypt.h>
+#include <beecrypt/sha1.h>
 
 #ifdef _REENTRANT
 # if WIN32
@@ -35,9 +36,6 @@
 # endif
 #endif
 
-#include "beecrypt.h"
-#include "sha1.h"
-
 #if (MP_WBITS == 64)
 # define FIPS186_STATE_SIZE	8
 #elif (MP_WBITS == 32)
Index: beecrypt-4.1.2/hmac.h
===================================================================
--- beecrypt-4.1.2.orig/hmac.h
+++ beecrypt-4.1.2/hmac.h
@@ -26,7 +26,7 @@
 #ifndef _HMAC_H
 #define _HMAC_H
 
-#include "beecrypt/beecrypt.h"
+#include <beecrypt/beecrypt.h>
 
 /*!\ingroup HMAC_m
  */
Index: beecrypt-4.1.2/hmacmd5.h
===================================================================
--- beecrypt-4.1.2.orig/hmacmd5.h
+++ beecrypt-4.1.2/hmacmd5.h
@@ -26,8 +26,8 @@
 #ifndef _HMACMD5_H
 #define _HMACMD5_H
 
-#include "beecrypt/hmac.h"
-#include "beecrypt/md5.h"
+#include <beecrypt/hmac.h>
+#include <beecrypt/md5.h>
 
 /*!\ingroup HMAC_md5_m
  */
Index: beecrypt-4.1.2/hmacsha1.h
===================================================================
--- beecrypt-4.1.2.orig/hmacsha1.h
+++ beecrypt-4.1.2/hmacsha1.h
@@ -26,8 +26,8 @@
 #ifndef _HMACSHA1_H
 #define _HMACSHA1_H
 
-#include "beecrypt/hmac.h"
-#include "beecrypt/sha1.h"
+#include <beecrypt/hmac.h>
+#include <beecrypt/sha1.h>
 
 /*!\ingroup HMAC_sha1_m
  */
Index: beecrypt-4.1.2/hmacsha256.h
===================================================================
--- beecrypt-4.1.2.orig/hmacsha256.h
+++ beecrypt-4.1.2/hmacsha256.h
@@ -26,8 +26,8 @@
 #ifndef _HMACSHA256_H
 #define _HMACSHA256_H
 
-#include "beecrypt/hmac.h"
-#include "beecrypt/sha256.h"
+#include <beecrypt/hmac.h>
+#include <beecrypt/sha256.h>
 
 /*!\ingroup HMAC_sha256_m
  */
Index: beecrypt-4.1.2/hmacsha384.h
===================================================================
--- beecrypt-4.1.2.orig/hmacsha384.h
+++ beecrypt-4.1.2/hmacsha384.h
@@ -26,8 +26,8 @@
 #ifndef _HMACSHA384_H
 #define _HMACSHA384_H
 
-#include "beecrypt/hmac.h"
-#include "beecrypt/sha384.h"
+#include <beecrypt/hmac.h>
+#include <beecrypt/sha384.h>
 
 /*!\ingroup HMAC_sha384_m
  */
Index: beecrypt-4.1.2/hmacsha512.h
===================================================================
--- beecrypt-4.1.2.orig/hmacsha512.h
+++ beecrypt-4.1.2/hmacsha512.h
@@ -26,8 +26,8 @@
 #ifndef _HMACSHA512_H
 #define _HMACSHA512_H
 
-#include "beecrypt/hmac.h"
-#include "beecrypt/sha512.h"
+#include <beecrypt/hmac.h>
+#include <beecrypt/sha512.h>
 
 /*!\ingroup HMAC_sha512_m
  */
Index: beecrypt-4.1.2/md5.h
===================================================================
--- beecrypt-4.1.2.orig/md5.h
+++ beecrypt-4.1.2/md5.h
@@ -26,7 +26,7 @@
 #ifndef _MD5_H
 #define _MD5_H
 
-#include "beecrypt/beecrypt.h"
+#include <beecrypt/beecrypt.h>
 
 /*!\brief Holds all the parameters necessary for the MD5 algorithm.
  * \ingroup HASH_md5_h
Index: beecrypt-4.1.2/memchunk.h
===================================================================
--- beecrypt-4.1.2.orig/memchunk.h
+++ beecrypt-4.1.2/memchunk.h
@@ -23,7 +23,7 @@
 #ifndef _MEMCHUNK_H
 #define _MEMCHUNK_H
 
-#include "beecrypt/api.h"
+#include <beecrypt/api.h>
 
 typedef struct
 {
Index: beecrypt-4.1.2/mp.h
===================================================================
--- beecrypt-4.1.2.orig/mp.h
+++ beecrypt-4.1.2/mp.h
@@ -41,8 +41,8 @@
 #ifndef _MP_H
 #define _MP_H
 
-#include "beecrypt/api.h"
-#include "beecrypt/mpopt.h"
+#include <beecrypt/api.h>
+#include <beecrypt/mpopt.h>
 
 #define MP_HWBITS	(MP_WBITS >> 1)
 #define MP_WBYTES	(MP_WBITS >> 3)
Index: beecrypt-4.1.2/mpbarrett.h
===================================================================
--- beecrypt-4.1.2.orig/mpbarrett.h
+++ beecrypt-4.1.2/mpbarrett.h
@@ -26,8 +26,8 @@
 #ifndef _MPBARRETT_H
 #define _MPBARRETT_H
 
-#include "beecrypt/beecrypt.h"
-#include "beecrypt/mpnumber.h"
+#include <beecrypt/beecrypt.h>
+#include <beecrypt/mpnumber.h>
 
 #ifdef __cplusplus
 # include <iostream>
Index: beecrypt-4.1.2/mpnumber.h
===================================================================
--- beecrypt-4.1.2.orig/mpnumber.h
+++ beecrypt-4.1.2/mpnumber.h
@@ -26,7 +26,7 @@
 #ifndef _MPNUMBER_H
 #define _MPNUMBER_H
 
-#include "beecrypt/mp.h"
+#include <beecrypt/mp.h>
 
 #ifdef __cplusplus
 # include <iostream>
Index: beecrypt-4.1.2/mpprime.h
===================================================================
--- beecrypt-4.1.2.orig/mpprime.h
+++ beecrypt-4.1.2/mpprime.h
@@ -26,7 +26,7 @@
 #ifndef _MPPRIME_H
 #define _MPPRIME_H
 
-#include "beecrypt/mpbarrett.h"
+#include <beecrypt/mpbarrett.h>
 
 #define SMALL_PRIMES_PRODUCT_MAX	32
 
Index: beecrypt-4.1.2/mtprng.h
===================================================================
--- beecrypt-4.1.2.orig/mtprng.h
+++ beecrypt-4.1.2/mtprng.h
@@ -26,7 +26,7 @@
 #ifndef _MTPRNG_H
 #define _MTPRNG_H
 
-#include "beecrypt/beecrypt.h"
+#include <beecrypt/beecrypt.h>
 
 #ifdef _REENTRANT
 # if WIN32
Index: beecrypt-4.1.2/pkcs1.h
===================================================================
--- beecrypt-4.1.2.orig/pkcs1.h
+++ beecrypt-4.1.2/pkcs1.h
@@ -6,7 +6,7 @@
 #ifndef _PKCS1_H
 #define _PKCS1_H
 
-#include "beecrypt/beecrypt.h"
+#include <beecrypt/beecrypt.h>
 
 #ifdef __cplusplus
 extern "C" {
Index: beecrypt-4.1.2/pkcs12.h
===================================================================
--- beecrypt-4.1.2.orig/pkcs12.h
+++ beecrypt-4.1.2/pkcs12.h
@@ -6,7 +6,7 @@
 #ifndef _PKCS12_H
 #define _PKCS12_H
 
-#include "beecrypt/beecrypt.h"
+#include <beecrypt/beecrypt.h>
 
 #ifdef __cplusplus
 extern "C" {
Index: beecrypt-4.1.2/rsa.h
===================================================================
--- beecrypt-4.1.2.orig/rsa.h
+++ beecrypt-4.1.2/rsa.h
@@ -26,7 +26,7 @@
 #ifndef _RSA_H
 #define _RSA_H
 
-#include "beecrypt/rsakp.h"
+#include <beecrypt/rsakp.h>
 
 #ifdef __cplusplus
 extern "C" {
Index: beecrypt-4.1.2/rsakp.h
===================================================================
--- beecrypt-4.1.2.orig/rsakp.h
+++ beecrypt-4.1.2/rsakp.h
@@ -26,7 +26,7 @@
 #ifndef _RSAKP_H
 #define _RSAKP_H
 
-#include "beecrypt/rsapk.h"
+#include <beecrypt/rsapk.h>
 
 /*!\brief RSA keypair.
  * \ingroup IF_rsa_m
Index: beecrypt-4.1.2/rsapk.h
===================================================================
--- beecrypt-4.1.2.orig/rsapk.h
+++ beecrypt-4.1.2/rsapk.h
@@ -26,7 +26,7 @@
 #ifndef _RSAPK_H
 #define _RSAPK_H
 
-#include "beecrypt/mpbarrett.h"
+#include <beecrypt/mpbarrett.h>
 
 #ifdef __cplusplus
 struct BEECRYPTAPI rsapk
Index: beecrypt-4.1.2/sha1.h
===================================================================
--- beecrypt-4.1.2.orig/sha1.h
+++ beecrypt-4.1.2/sha1.h
@@ -26,8 +26,8 @@
 #ifndef _SHA1_H
 #define _SHA1_H
 
-#include "beecrypt/beecrypt.h"
-#include "beecrypt/sha1opt.h"
+#include <beecrypt/beecrypt.h>
+#include <beecrypt/sha1opt.h>
 
 /*!\brief Holds all the parameters necessary for the SHA-1 algorithm.
  * \ingroup HASH_sha1_m
Index: beecrypt-4.1.2/sha1opt.h
===================================================================
--- beecrypt-4.1.2.orig/sha1opt.h
+++ beecrypt-4.1.2/sha1opt.h
@@ -26,8 +26,8 @@
 #ifndef _SHA1OPT_H
 #define _SHA1OPT_H
 
-#include "beecrypt/beecrypt.h"
-#include "beecrypt/sha1.h"
+#include <beecrypt/beecrypt.h>
+#include <beecrypt/sha1.h>
 
 #ifdef __cplusplus
 extern "C" {
Index: beecrypt-4.1.2/sha256.h
===================================================================
--- beecrypt-4.1.2.orig/sha256.h
+++ beecrypt-4.1.2/sha256.h
@@ -26,7 +26,7 @@
 #ifndef _SHA256_H
 #define _SHA256_H
 
-#include "beecrypt/beecrypt.h"
+#include <beecrypt/beecrypt.h>
 
 /*!\brief Holds all the parameters necessary for the SHA-256 algorithm.
  * \ingroup HASH_sha256_m
Index: beecrypt-4.1.2/sha384.h
===================================================================
--- beecrypt-4.1.2.orig/sha384.h
+++ beecrypt-4.1.2/sha384.h
@@ -26,7 +26,7 @@
 #ifndef _SHA384_H
 #define _SHA384_H
 
-#include "beecrypt/beecrypt.h"
+#include <beecrypt/beecrypt.h>
 
 /*!\brief Holds all the parameters necessary for the SHA-384 algorithm.
  * \ingroup HASH_sha384_m
Index: beecrypt-4.1.2/sha512.h
===================================================================
--- beecrypt-4.1.2.orig/sha512.h
+++ beecrypt-4.1.2/sha512.h
@@ -26,7 +26,7 @@
 #ifndef _SHA512_H
 #define _SHA512_H
 
-#include "beecrypt/beecrypt.h"
+#include <beecrypt/beecrypt.h>
 
 /*!\brief Holds all the parameters necessary for the SHA-512 algorithm.
  * \ingroup HASH_sha512_m
Index: beecrypt-4.1.2/sha_k.h
===================================================================
--- beecrypt-4.1.2.orig/sha_k.h
+++ beecrypt-4.1.2/sha_k.h
@@ -26,7 +26,7 @@
 #ifndef _SHA_K_H
 #define _SHA_K_H
 
-#include "beecrypt/beecrypt.h"
+#include <beecrypt/beecrypt.h>
 
 #ifdef __cplusplus
 extern "C" {
Index: beecrypt-4.1.2/timestamp.h
===================================================================
--- beecrypt-4.1.2.orig/timestamp.h
+++ beecrypt-4.1.2/timestamp.h
@@ -26,7 +26,7 @@
 #ifndef _TIMESTAMP_H
 #define _TIMESTAMP_H
 
-#include "beecrypt/beecrypt.h"
+#include <beecrypt/beecrypt.h>
 
 #if HAVE_LONG_LONG && !defined(__cplusplus) /* C++ doesn't like LL constants */
 # define ONE_SECOND	1000LL
